### DISCUSSION TITLE
Group devices, entities and entries

### DISCUSSION THREAD

**Discussion by herledant** [home-assistant/frontend#17482] [Automations / scripts / scenes] [answered]
Hello,

I wondering if it won't be a good idea to let user make group of entities, entries and device to find more easily what they looking for.

Sometime you create several entries for one automation and it could be great to be able to rearange them by this way.

Regards

**Comment by johntdyer** [comment, marked as answer]
I think labels support will help here.... I guess its coming but just blocked by the UX team at the moment 

https://github.com/home-assistant/frontend/pull/16626
https://github.com/home-assistant/core/pull/69996
https://github.com/home-assistant/home-assistant-js-websocket/pull/383
